shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -738,6 +749,14 @@ bool PlatformViewAndroid::Register(JNIEnv* env) {
     return false;
   }
 
+  g_create_overlay_surface_method = env->GetMethodID(
```

Comment:
sorry, I didn't catch this one: `g_create_overlay_surface_method` is being set multiple times.